src/test/integration/integrationtests.py

```diff
@@ -221,11 +262,12 @@ def test_ssl_v3(self):
             self.skipTest('SSLv3 support not available')
         self.assertFalse(check_ssl(ssl.PROTOCOL_SSLv3, locations_url, access_token))
 
+
```

Comment:
extra blank line?

Reply:
hmm, in pep8/flake8, we generally use two blank lines between two def sections. See error code E305 here: https://github.com/PyCQA/pycodestyle/blob/master/docs/intro.rst#error-codes